.coveragerc

```diff
@@ -243,6 +243,7 @@ omit =
     homeassistant/components/binary_sensor/ping.py
     homeassistant/components/binary_sensor/rest.py
     homeassistant/components/binary_sensor/tapsaff.py
+    homeassistant/components/binary_sensor/tesla.py
```

Comment:
Component with platforms have a group in the beginning of the file. Put an entry like this at the correct alphabetical place somewhere around line 170 where the t:s are:
```
homeassistant/components/tesla.py
homeassistant/components/*/tesla.py
```